models/public/brain-tumor-segmentation-0002/brain-tumor-segmentation-0002.md

```diff
@@ -67,55 +77,32 @@ The channels are ordered as `T1`, `T1ce`, `T2`, `Flair`.
 
 ## Output
 
-### Original model
+### Original Model
 
-Probabilities of the given voxel to be in the corresponding class, name - `304`, shape - `1,3,128,128,128`, output data format is `B,C,D,H,W` where:
+Probabilities of the given voxel to be in the corresponding class, name - `304`, shape - `1,3,128,128,128`, output data format is `B,C,D,H,W`, where:
 
 - `B` - batch size
 - `C` - channel
 - `D` - depth
 - `H` - height
 - `W` - width
 
-With the following channels: `whole tumor`, `tumor core` and `enhancing tumor`.
+The channels are ordered as `whole tumor`, `tumor core`, and `enhancing tumor`.
 
-### Converted model
+### Converted Model
 
-Probabilities of the given voxel to be in the corresponding class, name - `304`, shape - `1,3,128,128,128`, output data format is `B,C,D,H,W` where:
+Probabilities of the given voxel to be in the corresponding class, name - `304`, shape - `1,3,128,128,128`, output data format is `B,C,D,H,W`, where:
 
 - `B` - batch size
 - `C` - channel
 - `D` - depth
 - `H` - height
 - `W` - width
 
-With the following channels: `whole tumor`, `tumor core` and `enhancing tumor`.
+The channels are ordered as `whole tumor`, `tumor core`, and `enhancing tumor`.
 
 ## Legal Information
 
-The original model is distributed under the following
-[license](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE):
-
-```
-The MIT License
-
-Copyright (c) 2019 Dmitrii Lachinov
-
-Permission is hereby granted, free of charge, to any person obtaining a copy
-of this software and associated documentation files (the "Software"), to deal
-in the Software without restriction, including without limitation the rights
-to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
-furnished to do so, subject to the following conditions:
-
-The above copyright notice and this permission notice shall be included in
-all copies or substantial portions of the Software.
+The original model is distributed under the
+[MIT License](https://raw.githubusercontent.com/lachinov/brats2019/master/LICENSE).
 
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
-```
```

Comment:
What's the motivation for this change? This is how the licenses are documented for all public models.

Reply:
Don't we usually provide the link only? The text here matches what's in the link.

Reply:
For each model, there is a link to the license and a copy of that license. In the cases where the license is short, it is embedded directly in the README for easy reference.

Reply:
Okay I'll fix that